docs/modules/trino/examples/usage-guide/opa-bundle-trino-cm-428.yaml

```diff
@@ -13,14 +13,24 @@ data:
 
     default allow = false
 
+    # Allow non-batched access
     allow {
       is_admin
     }
+    # Allow batched access
     extended[i] {
       some i
       input.action.filterResources[i]
       is_admin
     }
+    # Corner case: filtering columns is done with a single table item, and many columns inside
+    extended[i] {
+      some i
+      input.action.operation == "FilterColumns"
+      count(input.action.filterResources) == 1
+      input.action.filterResources[0].table.columns[i]
+      is_admin
+    }
```

Comment:
Could you please add some explanatory text on the security page about the rules? Are these all the rule sets that are possible? What does the `input.action` struct look like?

Reply:
The problem I see is that we have not reached a stable API until https://github.com/trinodb/trino/pull/19532 is merged. I would therefore like to push back documenting the new OPA API until we have the stable API.
Anyway, Pablo is also adding some docs in the mentioned PR and will create a follow-up with even more docs IIRC. Once all this is available we should link to these docs

Reply:
Okay, sounds good to me!